### PR TITLE
Add api to vote/get theme votes (decorator not tested don't kill me)

### PR DESCRIPTION
**Pull request by dabreadman**
## Purpose

Addressed TPY-29, adding API to interact with Firestore to upvote/downvote themes.

## Proposed Change

Introduced helper function to upvote/downvote

## Checklist

- [x] A late-night coffee
- [x] A quick Medium article
- [x] Typing issue even though it's just copypasta
- [x] Frankensteining several articles and documentations
- [ ] Test if it works
- [x] Upvote working
- [x] Downvote working
- [x] Get all votes working
- [x] Get specific theme working
- [ ] Test decorator
- [ ] Get firebase-admin working to reduce latency from having to fetch old vote and add one

## Additional

Do you know that [`firebase-admin`](https://firebase.google.com/docs/admin/setup) exists and should be placed on server? I do, but I don't care :/

### CODE REVIEW

File: constants/ProjectThemes.ts [hunk 1]
```diff
@@ -4,6 +4,16 @@ export interface IProjectTheme {
   name: string
 }
 
+export enum Themes {
+  VANGOUGH,
```
Comment (dabreadman) [inline constants/ProjectThemes.ts:8]: Could use these `ENUMS` for themes, or 1,2,3,4 ...

Change the type for method when that happens, `string` is there for `test`ing

Comment (DanielNugent) [reply constants/ProjectThemes.ts:8]: yes makes sense

**Review by DanielNugent — APPROVED** [review]

File: constants/ProjectThemes.ts (hunk 1 above)
Comment (dabreadman) [reply constants/ProjectThemes.ts:8]: Feel free to resolve this and get it merged, wouldn't have time to work on this today

**Review by lexesj — APPROVED** [review]
LGTM!

Will make use of the enum on the backend too once this is merged in.

**Pull request merged**